hw/top_earlgrey/data/top_earlgrey.hjson

```diff
@@ -339,6 +328,16 @@
       reset_connections: {rst_ni: "spi_host1"},
       base_addr: "0x40310000",
     },
+    { name: "usbdev",
+      type: "usbdev",
+      clock_srcs: {clk_i: "usb", clk_aon_i: "aon", clk_usb_48mhz_i: "usb"},
```

Comment:
ah i see, so this is the part that will change later?

Reply:
Yeah, we'll remove the unnecessary second clock and some of the manually-instantiated CDC bits in usbdev.sv.